apps/backend/test/e2e/adoptions.e2e-spec.ts

```diff
@@ -21,6 +21,8 @@ import { AuthService } from '../../src/auth/auth.service';
 import { ConfigModule } from '@nestjs/config';
 import { getCoordinatesFromUser } from '../../src/user/user.helpers';
 
+Date.now = () => 1000;
```

Comment:
Is this change intentional?

Reply:
It was a quick test and I forgot to remove. Thanks